Resources/public/js/views/ez-navigationhubview.js

```diff
@@ -182,6 +231,21 @@ YUI.add('ez-navigationhubview', function (Y) {
         },
 
         /**
+         * Sets the active flag on the navigation item views
+         *
+         * @protected
+         * @method _setNavigationItemActive
+         * @param {Boolean} active
+         */
+        _setNavigationItemActive: function (active) {
+            Y.Object.each(this.get('zones'), function (zone, key) {
+                Y.Array.each(this.get(key + 'NavigationItems'), function (view) {
```

Comment:
The same as there: https://github.com/ezsystems/PlatformUIBundle/pull/162/files#r24313897